llvm/utils/TableGen/VTEmitter.cpp

```diff
@@ -27,6 +27,47 @@ class VTEmitter {
   void run(raw_ostream &OS);
 };
 
+static void VTtoGetLLVMTyString(raw_ostream &OS, const Record *VT) {
+  bool IsVector = VT->getValueAsBit("isVector");
+  if (IsVector)
+    OS << (VT->getValueAsBit("isScalable") ? "Scalable" : "Fixed")
+       << "VectorType::get(";
+
+  auto OutputVT = IsVector ? VT->getValueAsDef("ElementType") : VT;
+  int64_t OutputVTSize = OutputVT->getValueAsInt("Size");
+
+  if (OutputVT->getValueAsBit("isFP")) {
+    StringRef FloatTy = "";
```

Comment:
You don't need to initialize this. It will default to a nullptr with size 0 which is the same as an empty string as far a StringRef is concerned.

Reply:
Fixed :)